test/functional/apps/home/_navigation.ts

```diff
@@ -14,11 +14,17 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'header', 'home', 'timePicker']);
   const appsMenu = getService('appsMenu');
   const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
 
   describe('Kibana browser back navigation should work', function describeIndexTests() {
     before(async () => {
-      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/discover');
       await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
+      await kibanaServer.uiSettings.replace({ defaultIndex: 'logstash-*' });
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
```

Comment:
minor request to also add this here.  I'm sure we don't do it everywhere but since there's a call to `.replace({ defaultIndex: 'logstash-*' });` it's probably best to clear it out too;
```
await kibanaServer.uiSettings.replace({});
```

Reply:
but now I'm also wondering if unloading the kbn_archive unloads the config saved object?  That's where the default index pattern is saved.  You  could check by looking in Advanced Settings after this test runs and see if it's set.

Reply:
Totally!

Reply:
Ok, so I've been having issues starting the ftr server, for the last few days.
That said, I was able to use the saved objects info cli call.
After dropping out most of the test, and leaving the before and after calls in place (I also added the ui settings replace you asked for above), I got this:

```
node scripts/saved_objs_info.js --esUrl http://elastic:changeme@localhost:9220 --soTypes
 succ
      ### Saved Object Types Count: 6
      [
        {
          doc_count: 5,
          key: 'canvas-workpad-template'
        },
        {
          doc_count: 1,
          key: 'apm-telemetry'
        },
        {
          doc_count: 1,
          key: 'config'
        },
        {
          doc_count: 1,
          key: 'core-usage-stats'
        },
        {
          doc_count: 1,
          key: 'event_loop_delays_daily'
        },
        {
          doc_count: 1,
          key: 'space'
        }
      ]
```

So, the config count so type count, is 1.

I think that means we are good.  Your thoughts sir? @LeeDr

Reply:
Also, in case you were wondering, when I run the call just after staring the ftr server, I get this:

```
node scripts/saved_objs_info.js --esUrl http://elastic:changeme@localhost:9220 --soTypes
 succ
      ### Saved Object Types Count: 4
      [
        {
          doc_count: 5,
          key: 'canvas-workpad-template'
        },
        {
          doc_count: 1,
          key: 'apm-telemetry'
        },
        {
          doc_count: 1,
          key: 'config'
        },
        {
          doc_count: 1,
          key: 'space'
        }
      ]
```